test/framework/src/main/java/org/elasticsearch/test/rest/yaml/section/ExecutableSection.java

```diff
@@ -26,15 +26,18 @@
 
 import java.io.IOException;
 import java.util.Arrays;
+import java.util.List;
+
+import static java.util.Collections.unmodifiableList;
 
 /**
  * Represents a test fragment that can be executed (e.g. api call, assertion)
  */
 public interface ExecutableSection {
     /**
-     * {@link NamedXContentRegistry} needed in the {@link XContentParser} before calling {@link ExecutableSection#parse(XContentParser)}.
+     * Default list of {@link ExecutableSection}s available for tests.
      */
-    NamedXContentRegistry XCONTENT_REGISTRY = new NamedXContentRegistry(Arrays.asList(
+    List<NamedXContentRegistry.Entry> DEFAULT_EXECUTABLE_CONTEXTS = unmodifiableList(Arrays.asList(
```

Comment:
I think this can be final

Reply:
I started out declaring it `public static final` out of habit but checkstyle failed because they are all forced on that field because it is in an interface.